src/apiNetworkProvider.ts

```diff
@@ -120,6 +120,15 @@ export class ApiNetworkProvider implements INetworkProvider {
         return response.txHash;
     }
 
+    async sendBulkTransaction(txs: ITransaction[]): Promise<string> {
```

Comment:
Perhaps the name `sendTransactions()` is a bit more suitable.

Reply:
Also, perhaps implement this as `this.backingProxyNetworkProvider.sendTransactions()`.